### ISSUE TITLE
Incorrect lap times in 2023, Round 9, Austrian GP

### ISSUE THREAD

**Issue by theOehrly** [jolpica/jolpica-f1#13]
Some lap times in the 2023 Austrian GP are incorrect. Specifically, the following combinations of driver/lap number have incorrect lap times:

```
alonso: 48
bottas: 46
de_vries: 46
gasly: 57
hamilton: 49
kevin_magnussen: 45
norris: 50
ocon: 46
piastri: 49
russell: 48
sainz: 54
stroll: 48
tsunoda: 37
zhou: 50
```

Notably, all these lap times are the personal best lap times of some other driver, and they were set on that exact lap.

#### Example
Alonso is listed with a lap time of "1:08.739" on lap 48. The correct time would be "1:09.634". But "1:08.739" was the fastest lap of Norris and Norris set that lap time on lap 48. Norris' lap time on lap 48 is correct, therefore, the lap time is duplicated, not swapped.

#### Guess
The fastest lap times are specially highlighted in the source PDF. I would assume that this might have been an error when parsing the PDF for old Ergast. The data likely was imported incorrectly from there in an old database dump.
![grafik](https://github.com/jolpica/jolpica-f1/assets/23384863/a101f6cb-cf5f-43f1-a71a-06b86bad6259)
What speaks against this theory is that this is correct in current production Ergast but it was seemingly never reported as an error there.

**Comment by jolpica** [comment]
I've found that this is because of incorrect data in the Ergast results table. 
In the new database scheme, we no longer duplicate laptimes and fastest laptimes, so we choose the fastest laptime instead of the laps table time when its available.

This query to the results endpoint for Fernando Alonso 
https://ergast.com/api/f1/2023/9/drivers/alonso/results.json
Returns:
```json
...
"FastestLap": {
  "rank": "4",
  "lap": "48",
  "Time": {
    "time": "1:08.739"
  },
...
```
Which is what is listed as Alonso's lap 48.

I'm not sure theres much that can be done about this until we are live and perform corrections of the Ergast data.
Could you see if you confirm these findings?

**Comment by theOehrly** [comment]
@jolpica I agree, that seems to be the problem

And to correct my previous statement, this is NOT fixed in Ergast currently. The lap times are correct on the ``/laps`` endpoint. But the fastest lap returned by the ``/results`` endpoint is incorrect.

I also agree that it is probably best to hold back with fixing this, until we are independent of Ergast and import our own data.

**Comment by jolpica** [comment]
This issue is blocked by getting the import erroring on this round https://github.com/jolpica/jolpica-f1-pdf/tree/main-data/json/2023/09-Austrian%20Grand%20Prix/race

**Comment by harningle** [comment]
> This issue is blocked by getting the import erroring on this round https://github.com/jolpica/jolpica-f1-pdf/tree/main-data/json/2023/09-Austrian%20Grand%20Prix/race

yes it's due to https://github.com/harningle/fia-doc/issues/26. Not yet fixed

**Comment by theOehrly** [comment]
@harningle you closed that issue so this might no longer be blocked?

**Comment by jolpica** [comment]
I've run the import script, seems to be fixed

https://github.com/jolpica/jolpica-f1-pdf/commit/8f251fb7300f7e7a429f5597bd114cb8963580d2
